src/components/m-dropdown/_template.js

```diff
@@ -2,67 +2,32 @@ import html from 'nanohtml';
 import raw from 'nanohtml/raw';
 import classnames from 'classnames';
 
-const arrowIcon = '<axa-icon icon="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
+const arrowIcon = '<axa-icon size="sm" icon="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
 
-const getItemValue = (itemValue, index) => itemValue === null || itemValue === undefined ? index : itemValue;
-
-const nativeSelect = ({ title, items, size, value }) => html`<div class="${classnames('m-dropdown__select-wrap', {
```

Comment:
@jackblackCH 
@LucaMele 
Please rethink this change.
With this update both are rendered, the native `<select>` and an enhanced js based dropdown.
The **original** idea was to show the native `<select>` at mobile only, though it was never finished...

Reply:
I saw that. IMO it's less complex, more readable and less expensive to render both and hide and show the enhanced version. 

Mobile First -> Take the native as a mobile first approach and enhance it with the list on bigger viewports. no need to rerender.

Reply:
mmm ok i need to see the code here in order to have an opinion.. i will get back on this

Reply:
Okay, as I commented at the related code above.
Whatever the teams sees fitting best will be done

Reply:
i checked the code and now i cleaner to look at but we render additional HTML that is not necessary, which could have a performance issue if code would grow a lot. I would prefer to revert as it was as is cleaner form the consumer of the dropdown (if a user instepcts the DOM he will debug over dead code, which might be confusing). Rather that we have more complexity then our consumers therefore agree with @AndyOGo

Reply:
@jackblackCH ok i got both arguments and it make sense to have native on mobile and enhanced on desktop. Thought that is not what it was intended and we need to leave the possibility to have both. Some desktop use cases might require just native as well. So i Suggest to have 2: Enhanced which on mobile is native automatically (as you already did @jackblackCH ) AND with a setting JUST native. Please also provide more examples including title (title is what is displayed at the beggining and IS NOT in the options. Therefore once you choose one option, title wont be visibile anymore. Also please add example on "Small" and "Big" sized dropdown)

Reply:
I'll re-add the `native` "only" prop discussed with @LucaMele

Reply:
It's changed. the `native` prop enforces the visibility of the native on all vp's